packages/pwa-module/view/frontend/layout/default.xml

```diff
@@ -1,13 +1,8 @@
 <?xml version='1.0'?>
-<page
-    xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance'
-    xsi:noNamespaceSchemaLocation='urn:magento:framework:View/Layout/etc/page_configuration.xsd'>
+<page xmlns:xsi='http://www.w3.org/2001/XMLSchema-instance' xsi:noNamespaceSchemaLocation='urn:magento:framework:View/Layout/etc/page_configuration.xsd'>
     <body>
         <referenceContainer name='root'>
-            <block
-                class='Magento\Pwa\Block\Bundle'
-                name='Magento_Pwa_Block_Bundle'
-            />
+            <block class='Magento\Pwa\Block\Bundle' name='Magento_Pwa_Block_Bundle' />
```

Comment:
We should use double quotes throughout XML files.

Reply:
Completely agree. Will improve PR soon.

Reply:
https://github.com/magento-research/pwa-studio/pull/105/commits/a996e4f6b1386151ea51deb446834302c42d63d3